guidelines/enhancement_template.md

```diff
@@ -313,13 +313,21 @@ Upgrade expectations:
 
 Downgrade expectations:
 - If an `N->N+1` upgrade fails mid-way through, or if the `N+1` cluster is
-  misbehaving, it should be possible for the user to rollback to `N`. It is
-  acceptable to require some documented manual steps in order to fully restore
-  the downgraded cluster to its previous state. Examples of acceptable steps
-  include:
-  - Deleting any CVO-managed resources added by the new version. The
-    CVO does not currently delete resources that no longer exist in
-    the target version.
+  misbehaving, users may wish to rollback to `N`.  We [do not support
+  rollbacks or downgrades of any kind][no-downgrades], but if structuring to
+  allow rollbacks is possible, that is a nice internal safety valve.
+  It is acceptable to:
+  * Irrecoverably break the cluster on downgrades.
+  * Lock up during downgrades.
+  * Allow for messy downgrades with documented manual workaround steps.
+  * Implement clean, beautiful automatic downgrades. With sub-categories for:
+    * Manually polishing the post-downgrade state, such as deleting
+      any resources which the `N+1` release created but which the `N`
+      release doesn't know to delete.
+    * Automatically removing any `N+1`-specific resources, for example
+      via [cluster-version operator deletion
+      manifests](../enhancements/update/object-removal-manifest-annotation.md)
+      in the `N` release payload.
```

Comment:
I would add that we still expect EP authors to discuss what the downgrade story is w/ respect to the enhancement.

if it's straight up not going to be possible, even with manual procedures, that may be a point of discussion before agreeing to the EP.

Reply:
I've pushed 2b93171a -> 0e843bb9 to break upgrades and downgrades into their own subsections, to make it harder to forget to talk about them both (thanks Markdown linter).  And I've added a new paragraph to say that while all of the downgrade experiences are acceptable, the enhancement proposal should describe the experience that's expected, and for the less-fun experiences, explain why smooth downgrades were not selected.